identity-wallet/Cargo.toml

```diff
@@ -6,12 +6,10 @@ rust-version = "1.75.0"
 
 [dependencies]
 tauri.workspace = true
-
-oid4vc-manager.workspace = true
-siopv2.workspace = true
-oid4vci.workspace = true
-oid4vc-core.workspace = true
-oid4vp.workspace = true
+oid4vc.workspace = true
```

Comment:
Should `identity-wallet` generally have workspace dependencies if it has a life on its own or manage its own versions?

Reply:
I suppose it makes dependency management easier while both crates still live in the same repository.

Reply:
Yes keep it as much in sync as possible. It easier to maintain